.github/workflows/operator-release-please.yml

```
@@ -53,4 +56,27 @@ jobs:
           GH_TOKEN: ${{ steps.get_github_app_token.outputs.token }}
         working-directory: "release"
         run: |
-          gh release edit "${{ needs.releasePlease.outputs.release_name }}" --draft=false --latest=false
\ No newline at end of file
+          gh release edit "${{ needs.releasePlease.outputs.release_name }}" --draft=false --latest=false
+  publishImages:
+    env:
+      BUILD_TIMEOUT: 60
+      IMAGE_PREFIX: "grafana"
+    needs:
+      - "publishRelease"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Set up QEMU"
```

Don't we need to checkout the code?
```suggestion
    steps:
      - uses: actions/checkout@v4
      - name: "Set up QEMU"
```

---

No, https://github.com/docker/build-push-action?tab=readme-ov-file#git-context